tests/scripts/create-dev-cluster.sh

```diff
@@ -147,41 +173,56 @@ while getopts ":hrmfd:" opt; do
 	d)
 	    ROOK_EXAMPLES_DIR="$OPTARG"
 	    ;;
+    y)
+	    minikube_driver="$OPTARG"
+	    ;;
 	\?)
-	    echo  "Invalid option: -$OPTARG" >&2
+	    echo  "Invalid option: -$opt" >&2
 	    show_usage
 	    exit 1
 	    ;;
 	:)
-	    echo "Option -$OPTARG requires an argument." >&2
+	    echo "Option -$opt requires an argument." >&2
```

Comment:
Please, keep the usage of `OPTARG` in these case entries as the variable holds the value of the unknown or the option that requieres the argument.